macro/run_simScript.py

```diff
@@ -115,6 +118,9 @@
 if options.genie:    simEngine = "Genie"
 if options.nuradio:  simEngine = "nuRadiography"
 if options.ntuple:   simEngine = "Ntuple"
+if options.alpaca:
+    options.inputFile= False
```

Comment:
There's a way to add mutually exclusive arguments in `argparse`. Maybe have a look in the `argparse` documentation to disable the `inputFile` option when `--ALPACA` is used?

Reply:
thank you for the advice.
It is also resolved.
However, it could be revised for every situation that inputFile is using.